met-web/src/components/survey/listing/index.tsx

```diff
@@ -44,6 +43,11 @@ const SurveyListing = () => {
         });
     };
 
+    const handleSearchBarInput = (engagementNameFilter: string) => {
```

Comment:
This defeats the purpose of having a click button for the search. Since a click button is part of the design, we will assume that that is the UX needed.  So we won't trigger handleSearchBarClick on text change. We will trigger it on button click

Reply:
removed